Tensile/SolutionLibrary.py

```diff
@@ -458,7 +458,7 @@ def remapSolutionIndicesStartingFrom(self, curIndex):
     def merge(self, other, startIndex=0):
         assert self.__class__ == other.__class__
 
-        curIndex = max(startIndex, max(self.solutions.keys()) + 1) if self.solutions else 0
+        curIndex = max(startIndex, max(self.solutions.keys()) + 1 if self.solutions else 0)
```

Comment:
If we make the change in this way, isn't self.solutions.keys()  called even if self.solutions?
Is that OK?